src/sample/stuDashController.java

```diff
@@ -104,11 +124,33 @@ private void ChoiceBoxItem() {
     programme.getItems().addAll("BSc Computer Science", "BSc Mathematics", "BSc Management");
     major.getItems().addAll("Software Engineering", "Intelligent Computing", "Computing Infrastructure");
     minor.getItems().addAll("Accounting", "Mathematics", "Electives");
+    
+
+  }
+
+  public void display() throws IOException{
+    File stuinfoFile = new File("data/Student Dashboard/"+name.getText()+".txt");
+    Scanner sc = new Scanner(stuinfoFile);
+    String name= sc.nextLine();
+    String ic = sc.nextLine();
+    while(sc.hasNextLine())
+    {
+      System.out.println(name);
+      System.out.println(ic);
+    }
+    sc.close();
   }
 
   @Override
-  public void initialize(URL location, ResourceBundle resources) {
+  public void initialize(URL location, ResourceBundle resources){
     ChoiceBoxItem();
-    defaultInfo();
+    try {
```

Comment:
Why try catch

Reply:
this one i fix according to vscode suggestion